dali/operators/reader/nvdecoder/sequencewrapper.h

```diff
@@ -97,26 +76,18 @@ struct SequenceWrapper {
 
   Tensor<GPUBackend> sequence;
 
-  int count;
-  int height;
-  int width;
-  int channels;
-  int label;
+  int count = -1;
+  int height = -1;
+  int width = -1;
+  int channels = -1;
+  int label = -1;
   vector<double> timestamps;
-  int first_frame_idx;
-  DALIDataType dtype;
+  int first_frame_idx = -1;
+  DALIDataType dtype = {};
   std::function<void(void)> read_sample_f;
 
  private:
-  void wait_until_started_() const {
-    std::unique_lock<std::mutex> lock{started_lock_};
-    started_cv_.wait(lock, [&](){return started_;});
-  }
-
-  mutable std::mutex started_lock_;
-  mutable std::condition_variable started_cv_;
-  cudaEvent_t event_;
-  bool started_;
+  CUDAEvent event_ = {};
```

Comment:
```suggestion
  CUDAEvent event_;
```
It has a default constructor.

Reply:
Done